README.md

```diff
@@ -1,4 +1,3 @@
-[![Build Status](https://travis-ci.com/IBM/openapi-validator.svg?branch=main)](https://travis-ci.com/IBM/openapi-validator)
```

Comment:
You could use this line instead for the build status:
```
[![Build Status](https://github.com/IBM/openapi-validator/actions/workflows/build.yaml/badge.svg)](https://github.com/IBM/openapi-validator/actions/workflows/build.yaml)
```

Reply:
Done!